pages/pipelines/command_step.md.erb

```diff
@@ -2,26 +2,113 @@
 
 <%= toc %>
 
-A *command* step runs a command on an agent. The following is the most simple *command* step:
+
+A *command* step runs a command on an agent. The command can be the name of a script, or <cli things>
+
+A basic command step can be defined in your pipeline settings, or in your [pipeline.yml](/docs/pipelines/uploading-pipelines) file.
+
+<%= image "command-step.png", width: 944/2, height: 364/2, alt: "Screenshot of a basic command step" %>
 
 ```yml
-- command: "command.sh"
+steps:
+  - command: "command.sh"
 ```
 
-The following step includes every configuration property for *command* steps:
+
+## Command Step Attributes
+
+_Optional attributes:_
+
+<table>
+  <tr>
+    <td><code>name</code></td>
+    <td>
+      The text and/or emoji that will be displayed in the pipeline visualisation in Buildkite.<br>
```

Comment:
Sounds like a plan. I was hesitant about label for a bit, but I'm coming around to it. I wanted to be super clear that it can be just emoji, or just text, or both, which I wasn't sure was super clear if we just say label? But I'm keen to test drive it and see how it goes.